### PR TITLE
chore/ci: move profiling to Actions

### PR DESCRIPTION
**Pull request by curbengh**
## What does it do?
Continue https://github.com/hexojs/hexo/pull/4525
Finally replace Travis.

## Pull request tasks

~~- [ ] Add test cases for the changes.~~
- [x] Passed the CI test.

### CODE REVIEW

**Comment by curbengh** [comment]
https://cc5083362a7a8f7b92347dfa81133c6b5ffd13b4-10-hexo.surge.sh/flamegraph.html
https://cc5083362a7a8f7b92347dfa81133c6b5ffd13b4-12-hexo.surge.sh/flamegraph.html
https://cc5083362a7a8f7b92347dfa81133c6b5ffd13b4-14-hexo.surge.sh/flamegraph.html

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: ['10.x', '12.x', '14.x']
+        node-version: ['10', '12', '14']
```
Comment (curbengh) [inline .github/workflows/benchmark.yml:19]: Surge.sh's subdomain cannot have a dot, `10` & `10.x` seem to produce similar result.

**Review by SukkaW — COMMENTED** [review]
What about making benchmark & profiling as two dedicated jobs?

**Review by SukkaW — APPROVED** [review]

**Comment by SukkaW** [comment]
image

@curbengh

**Pull request merged**